### PR TITLE
Add spanish translation

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
I started a glossaire to note references of the translation
https://docs.google.com/spreadsheets/d/1DwJD0DQO6VihpjE1zHLjy25hO06pa78flPE12I1QY6E/edit#gid=0

### CODE REVIEW

File: projects/arlas-components/assets/i18n/es.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
+	"BBox generator": "Generar un cuadro delimitador",
+	"First corner": "Primera esquina",
+	"Second corner": "Segunda esquina",
+	"Close": "Cerrar detalle",
```
Comment (QuCMGisaia) [inline projects/arlas-components/assets/i18n/es.json:5]: Are we sure the close key is only used for details ?

File: projects/arlas-components/assets/i18n/es.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	"Download": "Descargar",
+	"Cloud": "Nube",
+	"country": "Pais",
+	"Click to show details": "Haga clic para mostrar los detalles",
+	"Geo-sort": "Geo-ordenación",
+	"Geo sort action": "Ordenar por distancia desde el centro del mapa",
+	"Timeout": "Tiempo de espera superado",
+	"Problem parsing input file": "Error al analizar el archivo de entrada",
+	"Only `zip` file is allowed": "Sólo se aceptan archivos `zip`",
+	"Zip file must contain at least a `*.shp`, `*.shx` and `*.dbf` or a `*.json`": "El archivo zip debe contener al menos un archivo `*.shp`, `*.shx` y `*.dbf` O un archivo `*.json`.",
```
Comment (QuCMGisaia) [inline projects/arlas-components/assets/i18n/es.json:77]: 0 instead of o

File: src/assets/i18n/es.json
Comment (QuCMGisaia) [inline src/assets/i18n/es.json:1]: Same comments on this file as the one in the projects/arlas-components

**Review by QuCMGisaia — APPROVED** [review]

**Review by sebbousquet — APPROVED** [review]

File: projects/arlas-components/assets/i18n/es.json (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-components/assets/i18n/es.json:5]: fixed

File: projects/arlas-components/assets/i18n/es.json (hunk 2 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-components/assets/i18n/es.json:80]: Te lo jure que es un O

**Pull request merged**